src/traits.rs

```diff
@@ -973,6 +973,19 @@ pub trait SystemExt: Sized + Debug + Default {
     /// ```
     fn get_version(&self) -> Option<String>;
 
+    #[cfg(any(target_os = "macos", target_os = "ios"))]
```

Comment:
There is no point into having a method on this trait specific to given targets, it has to support all targets.

Reply:
fixed in d3d6a3a